### DISCUSSION TITLE
About the verification of education mailbox

### DISCUSSION THREAD

**Discussion by 0Evo** [community/community#58507] [GitHub Education]
### Hi everyone,

![(POJR94N`T`1@YHN6BEAKQ4](https://github.com/community/community/assets/102132588/ebcfb0da-da3f-4475-8f51-07db2de62f48)
I am studying at Quanzhou Normal University，This school does not provide education mailboxes for students, but I still want to apply for the github education package as a student of this school

**Comment by the-officialjosh** [comment]
In my experience, at my school, we were not normally provided with education mailboxes. However, I was able to resolve this issue by reaching out to the IT department of my institution. They were able to provide me with a temporary mailbox or alternative solution. I would suggest trying the same approach at your school. Contact the IT department and explain your situation, expressing your interest in applying for the GitHub Education package. Once you receive their response, we can explore other alternative solutions if necessary. I hope they are able to assist you in resolving this matter.
